terminologies/technique/literatureMinning.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/technique/literatureMinning",

```

Comment:
"literatureMining" not minning; is a consistent typo and needs to be corrected in several places in the schema